flaky-tests/ci-providers/droneci.md

```diff
@@ -0,0 +1,151 @@
+---
+description: Configure Flaky Tests using Drone CI
+---
+
+# Drone CI
+
+## Getting Started
+
+You can use the Flaky Tests CLI within your [Drone CI](https://www.drone.io/) pipelines to upload and analyze your test results.
+
+{% hint style="info" %}
+The Trunk Flaky Tests CLI currently only supports x86_64 and arm64 for both Linux and macOS. If you have another use case, please get in touch with support at [https://slack.trunk.io](https://slack.trunk.io/). For the best results, you'll need to validate that your test invocation does not use cached test results and does not automatically retry failing tests.
+{% endhint %}
+
+### Create a Drone CI Pipeline
+
+Create a Drone CI pipeline (or modify an existing one) to run the tests that you want to monitor. The pipeline should produce a test report in [**JUnit XML**](https://github.com/testmoapp/junitxml) format. Most testing frameworks support XML output. See [Testing Framework Configuration](../frameworks/) for guides for common testing frameworks. Make sure that your test invocation doesn't use cached test results, and doesn't automatically retry failing tests.
```

Comment:
Does the `../frameworks/` relative link work?

Reply:
(doesn't in the GitBooks preview

Reply:
I fixed it.